netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -175,8 +177,18 @@ public NettyChannelBuilder eventLoopGroup(@Nullable EventLoopGroup eventLoopGrou
   }
 
   /**
+   * Trust Store to be used instead of system default.
+   * @throws SSLException
+   */
+  @Override
+  public NettyChannelBuilder trustStore(File trustCertCollectionFile) throws SSLException {
+    return sslContext(GrpcSslContexts.forClient().trustManager(trustCertCollectionFile).build());
```

Comment:
May have to add `keyManager` also here. But am waiting for your approval on the pattern. then will add support `keyManager` if needed

Reply:
I'm not excited that this builds the sslContext all at once, as it means it can't be combined with any other TLS configuration (like keyManager). Although I'm also not excited about adding keyManager to ManagedChannelBuilder; it's a fairly advanced use case and needs extra configuration on server-side as well (like request/require client certs and trust store). Going down this road feels like we end up re-inventing our own TLS context for configuration.

I think I'd be happier with a `useTransportSecurity(File trustCerts)`. But I have reservations about it as well as I feel it is dangerously close to opening a can of worms for other TLS configuration. And at the same time I'm not too bothered by the status quo of needing to use a specific builder when managing your own CA.

Reply:
Ok. Should I ignore this then?